header.php

```diff
@@ -126,7 +126,7 @@
 							/** Loading WordPress Custom Menu with Fallback to wp_list_pages **/
 							wp_nav_menu( array( 
 								'menu' => 'main-nav', 
-								'items_wrap'      => '<ul id="%1$s" class="%2$s" role="navigation">%3$s</ul>',
+								'items_wrap'      => '<nav aria-labelledby="main-nav"><ul id="%1$s" class="%2$s" role="navigation">%3$s</ul></nav>',
```

Comment:
This is also incorrect use of aria-labelledby - `#main-nav` is not an element that provides a label.

Reply:
Fixed in a commit